python/pyspark/cloudpickle.py

```diff
@@ -802,9 +802,8 @@ def save_not_implemented(self, obj):
         self.save_reduce(_gen_not_implemented, ())
 
     if PY3:
-        dispatch[io.TextIOWrapper] = save_file
-    else:
-        dispatch[file] = save_file
+        file = io.TextIOWrapper
+    dispatch[file] = save_file
```

Comment:
I think this one is actually related with cloudpickle's PR.

I was trying to (exactly) match this file to a specific version of cloudpickle (which is currently 0.4.3 - https://github.com/cloudpipe/cloudpickle/releases/tag/v0.4.3). So, I thought we could wait for more feedback there. 

At least, I was thinking that we should match it to https://github.com/cloudpipe/cloudpickle/tree/0.4.x If that one is merged, I could backport that change into 0.4.x branch.

Reply:
This was merged upstream.